WalletWasabi.Documentation/FAQ.md

```diff
@@ -0,0 +1,178 @@
+
+### What is the Anonymity Set?
+
+The anonymity set is effectively the size of the group you are hiding in. 
+
+If 3 people take part in a CoinJoin (with equal size inputs) and there are 3 outputs then each of those output coins has an anonymity set of 3.
+
+```
+0.1 BTC (Alice)       0.1 BTC (Anon set 3)
+0.3 BTC (Bob) 	  ->  0.1 BTC (Anon set 3)
+0.4 BTC (Charlie)     0.1 BTC (Anon set 3)
+                      0.2 BTC (Change Coin Bob)
+                      0.3 BTC (Change Coin Charlie)
+```
+
+There is no way to know which of the anon set output coins are owned by which of the input owners.
+All an observer knows is that a specific anon set output coin is owned by one of the owners of one of the input Coins i.e. 3 people - hence an anonymity set of 3.
+
+### How Do I change the default number of mixing rounds (the Anonymity Set)?
+
+Go to File>Open>Config and change the value of **N** in "MixUntilAnonymitySet": **N** to the Anonymity Set you desire. More is better, (arguably only up to a point). Remember that you pay a fee proportional to the Anonymity Set.
+
+### Can I mix more than the round's minimum? ###
+
+Yes. 
+In a round with a ~0.1 BTC minimum, you could mix ~ 0.3 BTC and get a ~ 0.1 BTC output & a ~ 0.2 BTC output.
+Similarly, with a 0.7 BTC input you would expect the following outputs: ~ 0.1, ~ 0.2, ~ 0.4 BTC. The possible values of equal output that can be created are 0.1 x 2^n where is a positive integer (or zero).
+
+### Why is the minimum mixing amount a weird number?
+
+The output value changes each round to ensure that you can que a coin and have it remix (mix over and over again - increasing the anonymity set, improving privacy). As a result the round mixing amount will often be a specific number which generally decreases as the rounds proceed, with a reset once a lower bound is reached. 
+
+# Post-Mix
+
+### What do I do now that I have mixed my coins?
+
+Wasabi is a very good wallet, and it is advisable to manage your funds using the wallet once you have mixed. If you are going to send your funds to another wallet (say, a mobile wallet for convenience) then there are a couple of important things to consider.
+- Unless the wallet has coin control you will merge your coins which damages privacy. 
+- Unless the wallet connects to your own full node you will leak information to the server supplying the blocks/filters.
+As a result, there are few wallets that are suitable. 
+
+Personally, I recommend sending your Zerolink Change to [Samourai Wallet](https://samouraiwallet.com/) or [Blockstream Green](https://blockstream.com/green/) for small-spends where you are less concerned about revealing any information. 
```

Comment:
Can you remove personal recommendations? If you want to recommend then maybe recommend ones those can connect to your own full node, but those are fairly impractical, so removing the specific recommendations and leave the general recommendations may be the best.

Reply:
You may want to also add a recommendations of trying to spend whole coins whenever possible.